applications/nrf_desktop/sample.yaml

```diff
@@ -1,6 +1,8 @@
 sample:
-  name: nRF Desktop
-  description: HID device reference design
+  name: nRF Desktop - HID reference design
+  description: The nRF Desktop is an HID reference design application.
+    It can be configured to function as a mouse, keyboard or USB dongle
+    bridging radio connected HID peripheral.
```

Comment:
I wonder a bit if we should use term `radio` or `Bluetooth LE` here (we do not support custom radio protocols in nRF Destkop at this point)

Reply:
In this case I meant radio to be understood as wireless. Anyway we know it is possible to use other protocols in the application. I would leave it as is.